### ISSUE TITLE
Android app crashes after updating RN to 0.45

### ISSUE THREAD

**Issue by iremlopsum** [philipphecht/react-native-doc-viewer#16]
I updated RN 0.45.0 and after that the app crashes as soon as I try to open any file. I found that If I change my targetSdkVersion to 23 (instead of 25) it doesn't crash, but instead tells me that "Media not found".

Any solution?

**Comment by philipphecht** [comment]
i dont have tested it on 0.45. i will test it and give you a feedback

**Comment by iremlopsum** [comment]
Okay. Figured it out. 

At the moment you are saving files to apps cache, but other apps cannot access those files. This is an Android this. They are pushing for devs to use content :// instead of file://. I run into this because I'm using Android 7.1.2. I made some modifications to your library and made it work. But because I defined [FileProvider](https://developer.android.com/reference/android/support/v4/content/FileProvider.html) in the manifest file of your library it breaks another library that uses FileProvider when merging manifest files. Not sure how to get around it yet. I can do a pull request with my solution.
